src/kili/entrypoints/mutations/asset/__init__.py

```diff
@@ -88,6 +92,16 @@ def append_many_to_dataset(
             disable_tqdm: If `True`, the progress bar will be disabled
             wait_until_availability: If `True`, the function will return once the assets are fully imported in Kili.
                 If `False`, the function will return faster but the assets might not be fully processed by the server.
+            from_csv: Path to a csv file containing the text assets to import.
+                Only used for `TEXT` projects. If provided, `content_array` must not be provided.
+            csv_separator: Separator used in the csv file. Only used if `from_csv` is provided.
+            csv_content_column: Name of the column containing the text assets to import.
+                Only used if `from_csv` is provided.
+                If None, the first column of the csv file will be used.
+            csv_external_id_column: Name of the column containing the external ids of the assets to import.
+                Only used if `from_csv` is provided.
+                If None, the external ids will be set to the passed `external_id_array` or to the csv row index.
```

Comment:
in order to avoid complexity, I think that we should force the user to give specific column name. On the app, we can import assets from url given in a csv and the column name should explicitely be `url` and `externalId' (https://docs.kili-technology.com/docs/adding-assets-to-project#required-csv-file-format)
I think that this is giving too much freedom for something for column names that will always be the same for the user and that will increase the complexity of the function and the code

Reply:
what do you think ?

Reply:
yep I agree

Reply:
I was thinking about removing completely `csv_external_id_column` and `csv_content_column`and to rather always search for the column named `content` and `externalId`in the csv